examples/api/lib/widgets/overlay/overlay.0.dart

```diff
@@ -0,0 +1,244 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for Overlay
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const OverlayApp());
+
+class OverlayApp extends StatelessWidget {
+  const OverlayApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: OverlayExample(),
+    );
+  }
+}
+
+class OverlayExample extends StatefulWidget {
+  const OverlayExample({super.key});
+
+  @override
+  State<OverlayExample> createState() => _OverlayExampleState();
+}
+
+class _OverlayExampleState extends State<OverlayExample> {
+  OverlayEntry? overlayEntry;
+  int currentPageIndex = 0;
+
+  void createHighlightOverlay({
+    required AlignmentGeometry alignment,
```

Comment:
This is going to point to the right section if the writing direction is RTL.

Reply:
Added `AlignmentDirectional` just in case, the example itself won't update tho in RTL.